bouncer/tests/all_concurrent_tests.ts

```diff
@@ -65,7 +63,6 @@ runWithTimeout(runAllConcurrentTests(), 2000000)
     process.exit(0);
   })
   .catch((error) => {
-    console.error!('All concurrent tests timed out. Exiting.');
```

Comment:
This error message was logging whenever an error happened, not just a timeout, so it was very misleading. We already print a timeout log message and print the error here otherwise, so this error message was redundant.